Dockerfile/dockbix-xxl-4.0/Dockerfile

```diff
@@ -97,7 +97,7 @@ COPY container-files-zabbix /
 RUN \
   export FPING_VERSION=4.0 && \
   yum clean all && \
-  yum install -y https://repo.zabbix.com/zabbix/3.5/rhel/7/x86_64/zabbix-release-3.5-1.el7.noarch.rpm && \
+  yum install -y https://repo.zabbix.com/zabbix/4.0/rhel/7/x86_64/zabbix-release-3.5-1.el7.noarch.rpm && \
```

Comment:
Can we use https://repo.zabbix.com/zabbix/4.0/rhel/7/x86_64/zabbix-release-4.0-2.el7.noarch.rpm here?

Reply:
I didn't want to change anything unnecessarily. I'll update the PR!